### ISSUE TITLE
Development Team Meeting Agenda

### ISSUE THREAD

**Issue by brandohayes** [hackforla/expunge-assist#411]
### **Overview**
This issue tracks the running agenda for the Development Team Weekly Meetings

### **Layout**
Agenda for XX/XX/XXXX

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.

**Comment by brandohayes** [comment]
### **Overview**
This issue tracks the running agenda for the Development Team Weekly Meetings

### **Layout**
Agenda for 01/06/2022

### Prework

### Roll Call

### Recurring items: 
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.

### New Action items
 - [ ] Complete Lettuce Meet Request to obtain team's availability for ALL Teams Meeting

### Notes:

- Daniel is working on merging new updates into the product from both him and Brad.
- The Dev team recruited new developers. They will invite them to the next meeting and point them to the onboarding meeting with Bonnie.

**Comment by thomasdemoner** [comment]
# Agenda for 01/13/22

### Items to address:

- [ ] Thomas
    - [ ] New devs? Introduce, address onboarding
    - [ ] Go over issues without project
    - [ ] Lead dev open role posting 

### Notes:

-

**Comment by brandohayes** [comment]
### Agenda for 01/20/2022

### Roll Call
- Daniel
- Brad
- Jen L
- Tom
- Brandon


### Recurring items: 
 - [x] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.

### Notes:
Daniel 
- Working on linking up with Content Writing (Sam) to review changes needed on the product

Jen
- Decided to join and will start picking up projects in the backlog
- Officially has been onboarded and has access to the Github page and Google Drive

Brad
- Finished task and his obtaining approval from Dev lead

Brandon
- Established new All Teams Meeting and has sent the invite to everyone on the Dev team

Tom
- Discussed Github Label changes required with Daniel
- Followed up on Dev Lead Job Req.

**Comment by brandohayes** [comment]
### Agenda for 01/27/2022

### Roll Call
- Dave
- Brad
- Igor (New Member)
- Tom
- Brandon
- Karsh (New Member)


### Recurring items: 
 - [x] review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
 - [ ] Accountability and Support Check.

### Notes:
Dave
- Requested another issue and will reach out to Daniel to obtain one.

Brad
- Working on the 'How E-A Works' section of the landing page. Needs answers from Design on if the correct image assets are in Figma as some images have an unremovable white background behind them. Brandon will find this out.

Brandon & Tom
- Gave a break down of the project to the new members and onboarded them.

Igor
 - A new member that is interested in during manual testing for the product

Karsh
- Interested in developing and has knowledge/experience with React. He also may be interested in other groups such as design and PM as well.

### New Action Items:
- Karsh and Igor will follow up with Daniel to introduce themself and talk more about the technical aspects of the project.
- Brandon will follow up with Design about the SVP image assets
- Dave will obtain an issue from Daniel.

**Comment by brandohayes** [comment]
### Agenda for 02/10/2022

### Roll Call
- Daisy
- Brad
- Daniel
- Tom
- Brandon


### Notes/Update:
Daisy
- Daisy reviewed the Figma design/current website and pointed out all of the issues that needed to be fixed (Refer to Figma for notes on issues)

Brad
- Currently working on “How Expunge Assist Works” Page with consist of three images

Daniel
- Fixed up page dials to match new design guideline.  
- Met with Sam about improvements such as unclear statements, double periods, and etc…..
- Daniel converted the branch to main instead of master
- New MVP Completion date is 02/28/2022. We plan to discuss the completion on the PM call that day.

Tom
- Reviewed labels with Dev


Brandon
- Discussed the onboarding/offboarding process/issue that need to be developed for the lead positions on GitHub

### New Action Items:
- Daniel will look at the request for a onboarding/offboarding procedure for a Dev Lead position on Github.

**Comment by thomasdemoner** [comment]
### Agenda for 03/10/2022

### Roll Call
- Daniel
- Tom
- Sydney
- Cristian Perpina


### Notes/Update:
Daisy
- Daisy reviewed the Figma design/current website and pointed out all of the issues that needed to be fixed (Refer to Figma for notes on issues)

Brad
- Currently working on “How Expunge Assist Works” Page with consist of three images

Daniel
- Fixed up page dials to match new design guideline.  
- Met with Sam about improvements such as unclear statements, double periods, and etc…..
- Daniel converted the branch to main instead of master
- New MVP Completion date is 02/28/2022. We plan to discuss the completion on the PM call that day.

Tom
- Reviewed labels with Dev


Brandon
- Discussed the onboarding/offboarding process/issue that need to be developed for the lead positions on GitHub

### New Action Items:
- Daniel will look at the request for a onboarding/offboarding procedure for a Dev Lead position on Github.

**Comment by thomasdemoner** [comment]
Notes from Monday 3/14

Attendance:
 - Thomas Demoner
 - Daniel Xiao
 - Daisy Yiu
 - Mahmoud A.
 - Olalekan Raheem

addressed:
 - Issue with welcome screen dialogue pop-up, as opposed to separate screens
 - Landing Page remake
 - Changing footer color
 - Lead Dev open role draft posted in a comment in our board's issue for recruiting devs.

**Comment by thomasdemoner** [comment]
notes from Thurs. 3/24

Attendance:
 - Thomas Demoner
 - Daniel Xiao
 - Sydney

Addressed:
- there's a welcome page now.
- Design and writing issues resolved; Sydney working on some dope adaptive design features.
- Remaining minimum for MVP release is getting the landing page done. Dan thinks he can make that happen by April 1st as well.

**Comment by sydneywalcoff** [comment]
# Agenda for 3/9/2023
## Roll Call
- Raman 
- Tom G
- Sydney
- Jamie Siu
- Thomas 
## Recurring items:
 review any issues that are in the [new issue approval column](https://github.com/hackforla/expunge-assist/projects/1#column-15701398)
## Accountability and Support Check.
## New Action items
## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 3/16/2023

note taker: @sydneywalcoff 

## Roll Call

- Sydney @sydneywalcoff 

## News/Updates

- need to create a new dev branch, but I want to get more info from daniel first
- holding up on reviews until I have clarification, may need to cherrypick PR commits into the new branch

## Recurring items:

- [ ] assign issues 
#697 

## Accountability and Support Check

- [ ] @ramankala working on #870 - submitted PR #882
- [ ] check PR #874 Contentful - @sydneywalcoff 
- [ ] @ramankala submitted PR #740 and @tglaz is reviewing
&nbsp;&nbsp;&nbsp;&nbsp; - changes requested might be related to the new dev branch

## New Action items

- [ ] 

## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 3/16/2023

note taker: @sydneywalcoff 

## Roll Call

- Sydney @sydneywalcoff 


## News/Updates

- Dev branch is fixed! Everyone is good to resume their work on issues!
- PR #740 was merged into Prod right before the meeting

## Recurring items:

- [ ] assign issues 


## Accountability and Support Check

- [ ] @ramankala working on #697 
- [ ] @sydneywalcoff to review  PR #882
&nbsp;&nbsp;&nbsp;&nbsp; - this looks good but I want to see if Daniel has other ideas before we make AppHistory obsolete
- [ ] check PR #874 Contentful - @sydneywalcoff 
- [x] get dev branch sorted out - @sydneywalcoff 

## New Action items



## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 4/20/2023

note taker: @sydneywalcoff 

## Roll Call

- Sydney @sydneywalcoff 



## Announcements

- happy break!
- merged to prod this afternoon, broke something, opened a PR
- remember to move issues you're working on to 'In Progress (active)' 
- we're getting very close to finishing IUT2 related tasks

## Recurring items:

- [ ] assign issues 
#691 

## Accountability and Support Check
open issues
- [ ] #703 @troymathis 
- [ ] #682 @tglaz 

open PRs
- [ ] #912 

other
- [ ] look into browser translations - @sydneywalcoff

## New Action items
- [ ]


## Notes:

- discuss the ability to maintain language file and the feasibility of keeping up with it; (contentful would be a good way to get it off the devs plate);

**Comment by sydneywalcoff** [comment]
# Agenda for 4/27/2023

note taker: @sydneywalcoff 

## Roll Call

- Sydney @sydneywalcoff 

## Announcements

- I'll have final confirmation on whether #690 will be part of IUT2 on Friday
- planning meeting next week; no checkin on thursday

## Recurring items:

- [ ] assign issues 
#691 

## Accountability and Support Check
open issues
- [ ] #703 @troymathis 
- [ ] 
open PRs
- [ ] #912 - @tglaz - waiting on confirmation from content about links;
- [ ] #917 - @tglaz - needs a review

other
- [ ] look into browser translations - @sydneywalcoff

## New Action items
- [ ] reach out to @troymathis - @sydneywalcoff 


## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 5/11/2023

note taker: @sydneywalcoff 

## Attendance

- Sydney @sydneywalcoff 


## Announcements

- pushed to production right before the meeting

## Recurring items:

- [ ] assign issues 
#691 
#923 

## Accountability and Support Check

### open issues
- [ ] #703 - @troymathis 
- [ ] #752 - @ramankala - working out some technical issue

### open PRs
- [ ] #912 - @tglaz - waiting on confirmation from content about links;

### other
- [ ] look into browser translations - @sydneywalcoff (after shipping IUT2)

## New Action items



## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 5/18/2023

note taker: @sydneywalcoff 

## Attendance

- Sydney @sydneywalcoff 
- 

## Announcements

- pushed to production right before the meeting
- 

## Recurring items:

- [ ] assign issues 
#691 - 
#923 - 

## Accountability and Support Check

### open issues
- [ ]

### open PRs
- [ ] #912 - @tglaz - waiting on confirmation from content about links;
- [ ] #703 @troymathis - last changes requested pls!

### other


## New Action items



## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 5/25/2023

note taker: 

## Attendance

- Sydney @sydneywalcoff 
- 

## Announcements

- pushed to production right before the meeting
- automation deleting dev branch after merging to production
- next week is planning week; no meetings!

## Recurring items:

- [ ] assign issues 


## Accountability and Support Check

### open issues
- [ ] nothing atm, 🎉 thanks for your PRs @AilinLuca @troymathis @tglaz 
- [ ] @thomasdemoner looking into the dev branch self-deletion 

### open PRs


### other


## New Action items



## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 6/8/2023

note taker: 

## Attendance

- Sydney @sydneywalcoff 


## Announcements

- pushed to production right before the meeting
- automation deleting dev branch after merging to production
- next week is planning week; no meetings!

## Recurring items:

- [ ] assign issues 
#690 - 
#945 - 
#955 - 

## Accountability and Support Check

### open issues
- [ ] #923 - @ramankala - eta 

### open PRs
- [ ] nothing atm 

### other


## New Action items



## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 6/8/2023

note taker: @sydneywalcoff 

## Attendance

- Sydney @sydneywalcoff 


## Announcements

- 

## Recurring items:

- [ ] assign issues 
#945 - 
#955 - 
#962 - 

## Accountability and Support Check

### open issues
- [ ] #923 - @ramankala - 
- [ ] #690 - @tglaz -

### open PRs
- [ ] nothing atm 

### other


## New Action items



## Notes:

-

**Comment by sydneywalcoff** [comment]
# Agenda for 6/29/2023

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Thomas


## Announcements/Discussion Points
- next week is planning week, no meeting

## Recurring items:

- [ ] assign issues 

#955 - 
#962 - 

## Accountability and Support Check

### open issues
- [ ] #923 - @sydneywalcoff - eta this weekend
- [ ] #690 - @tglaz - eta ?
- [ ] #945 - @ramankala 
- [ ] #966 - @lmitch10 & @sydneywalcoff -

### open PRs
- [ ] nothing atm 

### other


## New Action items



## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 7/13/2023

note taker: Lane Mitchell

## Attendance

- [x] Sydney
- [ ] Lane
- [ ] Thomas
- [ ] Tom 
- [ ] Raman

## Announcements/Discussion Points
- Bonnie did a test session of the site there should be incoming tickets as a result
- Team planning meeting was focused on the 'whats next' and around updating the mission statement for EA

## Recurring items:

- [ ] assign issues 

#955 - 
#962 - 

## Accountability and Support Check

### open issues
- [ ] #923 - @sydneywalcoff - eta
- [ ] #945 - @ramankala  - eta

### open PRs
- [ ] #968 - @tglaz - download letter as docx 

### other


## New Action items



## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 7/27/2023

note taker: Sydney Walcoff

## Attendance

- [x] Sydney
- [ ] Lane
- [ ] Thomas
- [ ] Tom 
- [ ] Raman

## Announcements/Discussion Points
- branch naming content issueNumber-description
- new [CONTRIBUTING.md](https://github.com/hackforla/expunge-assist/blob/main/CONTRIBUTING.md) includes process to write and review PRs;

## Recurring items:

- [x] assign issues 

#955 - 
#962 - 

## Accountability and Support Check

### open issues
- [x] #945 - @ramankala  - eta - next week

### open PRs
- [ ] #923 - @sydneywalcoff - shift bug

### other


## New Action items



## Notes:

**Comment by lmitch10** [comment]
# Agenda for 8/3/23

note taker: Lane Mitchell

## Attendance

- [ ] Sydney
- [ ]Lane
- [ ] Thomas
- [ ] Tom 
- [ ] Raman

## Announcements/Discussion Points
- branch naming content issueNumber-description
- new [CONTRIBUTING.md](https://github.com/hackforla/expunge-assist/blob/main/CONTRIBUTING.md) includes process to write and review PRs;

## Recurring items:

- [ ] assign issues 

#955 - @ramankala 
#962 - 

## Accountability and Support Check

### open issues
- [ ] #945 - @ramankala  - need more details

### open PRs
- [ ] #923 - @sydneywalcoff - shift bug - still appearing on popups

### other


## New Action items



## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 8/10/23

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Tom 
- [ ] Raman

## Announcements/Discussion Points

- Welcome new devs

## Recurring items:

- [ ] assign issues 
#962 
#945
#987 
#986

## Accountability and Support Check

### open issues
- [ ] #955 - @ramankala -
- [ ] #866 - @sydneywalcoff  - updated contributing.md, need to take a look at onboarding for devs

### open PRs
- [ ] #923  - @sydneywalcoff - still working on this

### other


## New Action items



## Notes:

**Comment by lmitch10** [comment]
# Agenda for 09/07/23

note taker: Lane Mitchell

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Tom 
- [ ] Raman
- [ ] Anthony

## Announcements/Discussion Points

## Recurring items:

### assign issues 

## Accountability and Support Check

### open issues


### open PRs
- [ ] #979 - @sydneywalcoff - need a new review

### other


## New Action items
- Project Manager discussion


## Notes:

**Comment by lmitch10** [comment]
# Agenda for 09/14/23

note taker: Lane Mitchell

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Tom 
- [ ] Raman
- [ ] Anthony
- [ ] Erin
- [ ] Hasham
- [ ] 
## Announcements/Discussion Points

- Introduce Hasham and Erin to the team! 
- Credits page discussion
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22)

## Recurring items:

### assign issues 
- [ ] #1001 

## Accountability and Support Check

### open issues
- [ ] #955 -
- [ ] #945 - 
- [ ] #1010 


### open PRs
- [ ] #979 -  still needs a review :(
- [ ]  #1005 - 
- [ ] #1006 -

### other


## New Action items


## Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 09/21/23

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Raman
- [ ] Whitney
- [ ] Hasham
- [ ] Anthony
- [ ] David
- [ ] Troy

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22)
- Thank you Whitney for writing issue #1026 🎉

## Recurring items:

- [ ] assign issues 
no available issues atm 🎉 


## Accountability and Support Check

### open issues
- [ ] #996 
- [ ] #945 
- [ ] #955 
- [ ] #1009 
- [ ] #1026

### open PRs
- [ ] #1027 - ready for review - 
- [ ] #1005 - still needs updates

### other


## New Action items



## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 09/28/23

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Raman
- [ ] Whitney
- [ ] David
- [ ] Hasham
- [ ] Troy

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22)
- not accepting any more devs at the moment

## Recurring items:

- [ ] assign issues 
no unassigned issues! 🎉 


## Accountability and Support Check

### open issues
- [ ] #996 
- [ ] #945 
- [ ] #955 
- [ ] #1026
- [ ] #1001

### open PRs
- [ ] #1009 
- [ ] #1008
- [ ] #1005

### other


## New Action items
- [ ] remove frontend dev posting


## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 10/12/23

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Raman
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David

## Announcements/Discussion Points

- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22)
- Coding Principles to keep in mind: maintainability, [responsiveness](https://chrome.google.com/webstore/detail/responsive-viewer/inmopeiepgfljkpkidclfgbgbmfcennb), and reusing code. 
- Thoughts on writing tests?

## Recurring items:

- [ ] assign issues 


## Accountability and Support Check

### open issues
- [ ] #945 
- [ ] #955 
- [ ] #996 

### open PRs
- [ ] #1034 - @whitneywind - left some comments
- [ ] #1026 - @davidwiese - I'll be deploying and testing this after the meeting
- [ ] #1008 - @hasham-qaiser - left some comments
- [ ] #1037  - @awellsbiz - testing this more after the meeting

### other
- Merged #1036 - thanks @awellsbiz 🎉 

## New Action items

- [ ] find a testing environment so we can create a QC process including other teams


## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 10/26/23

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Raman
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David

## Announcements/Discussion Points

- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22)
- If you weren't at the last meeting, check out the previous agenda for a cool extension to help with coding responsively
- 

## Recurring items:

- [ ] assign issues 
#945


## Accountability and Support Check

### open issues
- [ ] #996 

### open PRs
- [ ] #1033 
- [ ] #1034
- [ ] #1037 
- [ ] #1040 

### other


## New Action items



## Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 1/11/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Raman
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22)
- Happy New Year 🎉 
- DevOps (Documentation and Process) Tickets in the backlog !!
- On issues, we're going to start giving `ETC`s or Estimated Time to Completion when you take an issue, and then weekly updates after
<img width="250" alt="Screen Shot 2024-01-08 at 6 19 41 PM" src="https://github.com/hackforla/expunge-assist/assets/75542938/59df5ba4-3576-4485-844e-49491ce9af6a">

## Recurring items:

**assign issues** 
- [ ] #1073
- [ ] #1077
- [ ] #1079
- [ ] #1081 
- [ ] #1082 
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22) 🎉 


## Accountability and Support Check

### open issues
- [ ] #1074 
- [ ] #996
- [ ] #1048 

### open PRs
- [ ] #1040 / #955 
- [ ] #1084  / #1001 

### previous action items
- [ ] finding a test environment

## New Action items
- [ ] finding a CDN


## Other Notes:

**Comment by sydneywalcoff** [comment]
# Agenda for 1/18/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Raman
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+label%3A%22role%3A+development%22+-label%3A%22lead%22)
- added dev labels to specify type of task in addition to 'role: development': 
    - `front-end`: type of development task relating to the website
    - `DevOps`: type of development task related to Process, Github Actions, and Deployment
    - `ready for dev lead`: task ready for dev lead to evaluate
    - `lead`: task for a team lead to complete

## Recurring items:

**assign issues** 
- [ ] #...

### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] 

## New Action items
- [ ] 

## Other Notes:
-

**Comment by hasham-qaiser** [comment]
### Meeting notes: 
-Anthony shared new feature for testing data/functionality 
-discussed open PR's 
-discussed possible group huddle for issue #1074 

> # Agenda for 1/18/24
> note taker:
> 
> ## Attendance
> * [x]  Sydney
> * [ ]  Lane
> * [ ]  Raman
> * [x]  Hasham
> * [ ]  Whitney
> * [x]  Anthony
> * [x]  David
> 
> ## Announcements/Discussion Points
> * [dev project board](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+label%3A%22role%3A+development%22+-label%3A%22lead%22)
> * added dev labels to specify type of task in addition to `role: development`:
>   
>   * `front-end`: type of development task relating to the website
>   * `DevOps`: type of development task related to workflows, Github Actions, and Deployment
>   * `ready for dev lead`: task ready for dev lead to evaluate
>   * `lead`: task for a team lead to complete
> * make sure you're not committing to `dev` or `main`
> * Dev State: Letter Generator Demo
> 
> ## Recurring items:
> **assign issues**
> 
> * [x]  [Remove scroll when pop-ups appear #1098](https://github.com/hackforla/expunge-assist/issues/1098)
> * [x]  [Write `How to Review Pull Requests` Wiki Page #1081](https://github.com/hackforla/expunge-assist/issues/1081)
> * [ ]  [Write `How to Create Pull Requests` Wiki Page #1082](https://github.com/hackforla/expunge-assist/issues/1082)
> * [ ]  [Create an issue form from our blank issue template #1086](https://github.com/hackforla/expunge-assist/issues/1086)
> * [ ]  [Create a `Dependency` Template #1091](https://github.com/hackforla/expunge-assist/issues/1091)
>   ... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22) 🎉
> 
> ### Accountability and Support Check
> **open issues**
> 
> * [ ]  [Keyboard users lose focus after popup #1074](https://github.com/hackforla/expunge-assist/issues/1074)
> * [ ]  [Write `How to Create Issues` Wiki Page #1083](https://github.com/hackforla/expunge-assist/issues/1083)
> * [ ]  [Improving Visuals #1079](https://github.com/hackforla/expunge-assist/issues/1079)
> 
> **open PRs**
> 
> * [ ]  [#1073- edit links to remove tags #1111](https://github.com/hackforla/expunge-assist/pull/1111) / [Internal Links go to home page #1073](https://github.com/hackforla/expunge-assist/issues/1073)
> * [ ]  [Before you begin 1077 #1110](https://github.com/hackforla/expunge-assist/pull/1110) / [Before you begin #1077](https://github.com/hackforla/expunge-assist/issues/1077)
> * [ ]  [#996-add last popup #1078](https://github.com/hackforla/expunge-assist/pull/1078) / [Add Pop-Up to last page of Letter Generator  #996](https://github.com/hackforla/expunge-assist/issues/996)
> * [ ]  [#955 - Improved the progress bar to match figma #1040](https://github.com/hackforla/expunge-assist/pull/1040) / [Progress Bar Styling #955](https://github.com/hackforla/expunge-assist/issues/955)
> * [ ]  [#1085-remove email and text from footer #1109](https://github.com/hackforla/expunge-assist/pull/1109) / [Remove email and "Expunge Assist" text from footer #1085](https://github.com/hackforla/expunge-assist/issues/1085)
> 
> **previous action items**
> 
> * [ ]  finding a test environment [Find a test environment #1112](https://github.com/hackforla/expunge-assist/issues/1112)
> * [ ]  finding a CDN [Find a CDN #1113](https://github.com/hackforla/expunge-assist/issues/1113)
> 
> ## New Action items
> * [ ]  ...
> 
> ## Other Notes:
> * ...

**Comment by sydneywalcoff** [comment]
# Agenda for 1/25/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Lane
- [ ] Raman
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22)
- branch security check

## Recurring items:

**assign issues** 
- [ ] #1082 
- [ ] #1086 
- [ ] #1091 
- [ ] #1097 
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22) 🎉 

### Accountability and Support Check

**open issues**
- [ ] #1074 
- [ ] #996 
- [ ] #955
- [ ] #1079 
- [ ] #1096
- [ ] #1081 

**open PRs**
- [ ] #1111  / #1073 
- [ ] #1110 / #1077
- [ ] #1078 / #996
- [ ] #1040 / #955 
- [ ] #1109 / #1085

**previous action items**
- [ ] 

## New Action items
- [ ] 

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 2/8/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22)
- Lane is leaving the project :( 
- run `npm i` after the you next pull from `dev`
- remember to use the `ready for dev lead` label when you need me to look at PRs

## Recurring items:

**assign issues** 
- [ ] #1121 
- [ ] #1151 
- [ ] #1097 
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22) 🎉 

### Accountability and Support Check

**open issues**
- [ ] #1074 -
- [ ] #1082 - 
- [ ] #1083 -
- [ ] #1081 - 
- [ ] #1079 -

**open PRs**
- [ ] #PR / #ISSUE
no PRs 🎉 

**previous action items**
- [ ] ...

## New Action items
- [ ] 

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 2/15/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22)
- 

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 2/22/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Anthony
- [ ] David
- [ ] Troy

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22)
- 

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] ...

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 3/14/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22)
- New Dev - Welcome Eileen! 🎉 

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3A%22lead%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for DATE

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- New Process! [Testing Github Actions Locally](https://github.com/hackforla/expunge-assist/wiki/Testing-Github-Actions-Locally)
- 

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] ...

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 3/27/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- 

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] ...

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for DATE

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- 

## Recurring items:

**assign issues** 
- [ ] #1238 
- [ ] #1015 
- [ ] #1224 
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1325 
- [ ] #1235 
- [ ] #1231 
- [ ] #1243

**open PRs**
- [ ] #1324 / #1235 

**previous action items**
- [x] none 🎉

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 4/17/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney - 🕐
- [ ] David
- [ ] Troy
- [ ] Eileen - 🕐

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- keep an eye out for the new bot when you are assigned an issue

## Recurring items:

**assign issues** 
- [ ] #1316
- [ ] #1176
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1238 
- [ ] #1015
- [ ] #1086
- [ ] #1243

**open PRs**
- [ ] #1340 / #1231

**previous action items**
- [ ] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 4/24/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- no meeting next week for the All Teams meeting ! 
- 

## Recurring items:

**assign issues** 
- [ ] #1086
- [ ] #1091
- [ ] #1106
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #1349 / #1176
- [ ] #1354 / #1243

**previous action items**
- [ ] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 4/15/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- please pick from the top of the prioritized backlog

## Recurring items:

**assign issues** 
- [ ] #1106 
- [ ] #1392 
- [ ] #1270 
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1015 
- [ ] #1376 
- [ ] #1138 

**open PRs**
- [ ] #1389 / #1377

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 5/22/24

note taker: 

## Attendance

- [x] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)

## Recurring items:

**assign issues** 
- [ ] #381 
- [ ] #1106 
- [ ] #1392 
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1015 
- [ ] #1363 
- [ ] #1396 

**open PRs**
- [ ] #1403 / #1376
- [ ] #1389 / #1377
- [ ] #1406 / #1138 

**previous action items**
- [x] none 🎉

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 4/29/24

note taker: @sydneywalcoff 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney - 🕐
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- Responsive design as a dev
   - [responsive viewer chrome extension](https://chromewebstore.google.com/detail/responsive-viewer/inmopeiepgfljkpkidclfgbgbmfcennb)

## Recurring items:

**assign issues** 
- [ ] #1270 
- [ ] #1268 
- [ ] #1249 
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #381 
- [ ] #1015 
- [ ] #1392

**open PRs**
- [ ] #1403 / #1376
- [ ] #1407 / #1363
- [ ] #1406 / #1138 
- [ ] #1409 / #1106

**previous action items**
- [ ] none 🎉

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 6/12/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney - 🕐 
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- Code Freeze and Bonnie Review! 

## Recurring items:

**assign issues** 
- [ ] #1438 
- [ ] #1248
- [ ] #1268
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1244 - David
- [ ] #1270 - Eileen

**open PRs**
- [ ] #1441 / #1418
- [ ] #1426 / #1392
- [ ] #1407 / #1363
- [ ] #1403 / #1376
- [ ] #1433 / #381
- [ ] #1436 / #1015

**previous action items**
- [ ] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 6/19/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney - 🕐 
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/1?filterQuery=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22)
- new project board! Some of the links are wrong! ^this link is updated
- new process re: dependabot alerts
   - dependabot now targets the `dependencies` branch
   - `dependencies` is a playground to address any changes the packages updates may need; 
      - for example, 'prettier' vs 'prettier/react'

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/hackforla/expunge-assist/projects/1?card_filter_query=label%3A%22role%3A+development%22+-label%3Alead+-label%3A%22issue+level+ii%3A+main%22+-label%3A%22issue+level+iii%3A+epic%22) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] ...

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 7/10/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- 

## Recurring items:

**assign issues** 
- [ ] #1248 
- [ ] #1268 
- [ ] #1247 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1244 
- [ ] #1270 

**open PRs**
- [ ] #1466 / #1249

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 8/24/2024

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] David
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- hold up on testing issues if you're getting warnings

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] ...

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 09/16/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- Node v20.16
- Migration
- [Team Charter](https://docs.google.com/document/d/1wdqbY_f67lsDrvIEnyupjWNonEXVTQqvjBGZtpV0EVU/edit#heading=h.i5upiimvcwgn)

## Recurring items:

**assign issues** 
- [ ] #1547 
- [ ] #1546 
- [ ] #1555
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] 

**open PRs**
- [ ] #1498 / #1421
- [ ] #1490 / #1247
- [ ] #1491 / #1245
- [ ] #1536/ #1488
- [ ] #1466 / #1249

**previous action items**
- [ ] ...

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 9/23/24

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- Disregard what I said about Maria joining

## Recurring items:

**assign issues** 
- [ ] #1555 
- [ ] #1248
- [ ] #1268 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1547 
- [ ] #1546 


**open PRs**
- [ ] #1491 / #1245
- [ ] #1490 / #1247
- [ ] #1466 / #1249 

**previous action items**
- [x] none

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for DATE

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- we have a new PM 🎉  welcome Rick!

## Recurring items:

**assign issues** 
- [ ] #1246 
- [ ] #1270 
- [ ] #1268 

... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1416 
- [ ] #1352

**open PRs**
- [x] none 🎉 

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for November 4, 2024

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- 

## Recurring items:

**assign issues** 
- [ ] #1625 
- [ ] #1194 
- [ ] #1268 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1246 

**open PRs**
- [ ] #1633 / #1352
- [ ] #1634 / #1270

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by rcpeng34** [comment]
@sydneywalcoff can we add two items to the agenda?
Signing the [Project Charger v1](https://docs.google.com/document/d/1wdqbY_f67lsDrvIEnyupjWNonEXVTQqvjBGZtpV0EVU/edit?tab=t.0#heading=h.o5zn67cjjc5e)

An update on plan to discuss milestones with Bonnie

**Comment by sydneywalcoff** [comment]
# Agenda for DATE

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- Sign the [Project Charter v1](https://docs.google.com/document/d/1wdqbY_f67lsDrvIEnyupjWNonEXVTQqvjBGZtpV0EVU/edit?tab=t.0#heading=h.o5zn67cjjc5e) if you haven't

## Recurring items:

**assign issues** 
- [ ] #1268 
- [ ] #1532
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1194 
- [ ] #1576 
- [ ] #1625 

**open PRs**
- [ ] #1644 / #1246

**previous action items**
- [ ] nada 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 11/18

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- 

## Recurring items:

**assign issues** 
- [ ] #1142
- [ ] #1648 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1194 
- [ ] #1416 
- [ ] #1576

**open PRs**
- [ ] #1644 / #1246 

**previous action items**
- [ ] nada 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 1/6/25

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- welcome back 🎉 
- fill out the [lettuce meet](https://lettucemeet.com/l/MADG9)

## Recurring items:

**assign issues** 
- [ ] #1648
- [ ] #1532 
- [ ] #1469
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1142 
- [ ] #1576 
- [ ] #1416

**open PRs**
- [x] none 🎉 

**previous action items**
- [x] none 🎉

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 1/13/25

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- [Lettuce Meet](https://lettucemeet.com/l/MADG9) for new All Teams please!
- Migration coming soon!
- Dependabot updates - Thinking about looping y'all in on reviewing those, but need to figure out how that would work.

## Recurring items:

**assign issues** 
- [ ] #1268 
- [ ] #1577
   * this HAS to be merged after #1268, but can be worked on concurrently
- [ ] #1532
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 

### Accountability and Support Check

**open issues**
- [ ] #1469 
- [ ] #1142 
- [ ] #1416 

**open PRs**
- [ ] #1663 / #1648 

**previous action items**
- [x] none :tada:

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 1/27

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- starting February All Teams is on Thurs @ 6pm PT
   - First week of the month is still just PMs and Leads 

## Recurring items:

**assign issues** 
- [ ] #...
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #...

**open PRs**
- [ ] #PR / #ISSUE

**previous action items**
- [ ] ...

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 2/3/25

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- this weeks All Teams is just PMs and Leads

## Recurring items:

**assign issues** 
- [ ] #1678 
- [ ] #1681 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1268 
- [ ] #1142 
- [ ] #1577 
- [ ] #1667 

**open PRs**
- [x] none 🎉 

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 2/10/25

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- PMs met with Bonnie on Friday: [Updated Priority List](https://docs.google.com/spreadsheets/d/1uajit1EJKIKGd6wqQz6XC-GTYTycDo1Ea-fzAiBxCqs/edit?gid=1066885190#gid=1066885190)

## Recurring items:

**assign issues** 
- [ ] #1678 
- [ ] #1508 
- [ ] #1681 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1667 
- [ ] #1577 
- [ ] #1268 
- [ ] #1142 

**open PRs**
- [x] none 🎉 

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 2/17/2025

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- Please come to the All Teams on Thursdays @ 6pm

## Recurring items:

**assign issues** 
- [ ] #1508 
- [ ] #1524 
- [ ] #1678
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1667 
- [ ] #1268 
- [ ] #1577 
- [ ] #1142 

**open PRs**
- [x] none 🎉 

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 3/3/2025

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- 

## Recurring items:

**assign issues** 
- [ ] #1508 
- [ ] #1524 
- [ ] #1678 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1268 
- [ ] #1577 
- [ ] #1142 
- [ ] #1143 

**open PRs**
- [x] none

**previous action items**
- [x] none

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 3/10/25

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- Migration on hold until we finish Stakeholder 2.0 issues

## Recurring items:

**assign issues** 
- [ ] #1384 
- [ ] #1688 
- [ ] #1678 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1142 
- [ ] #1268 
- [ ] #1143
- [ ] #1508 

**open PRs**
- [x] none 🎉 

**previous action items**
- [x] look into Migration branch

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 3/17/25 ☘ 

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- need to move the [meeting time](https://lettucemeet.com/l/4mlMj)

## Recurring items:

**assign issues** 
- [ ] #1688 
- [ ] #1524 
- [ ] #1711 
... more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1384 
- [ ] #1508 
- [ ] #1143 
- [ ] #1268 
- [ ] #1142 

**open PRs**
- [x] none

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-

**Comment by sydneywalcoff** [comment]
# Agenda for 3/24/25 

note taker: 

## Attendance

- [ ] Sydney
- [ ] Hasham
- [ ] Whitney
- [ ] Troy
- [ ] Eileen
- [ ] Rick

## Announcements/Discussion Points
- [dev project board](https://github.com/orgs/hackforla/projects/77/views/4)
- need to move the [meeting time](https://lettucemeet.com/l/4mlMj)

## Recurring items:

**assign issues** 
- [ ] #1524
- [ ] #1711 
- [ ] #1693 
... and more on the [dev project board](https://github.com/orgs/hackforla/projects/77/views/4) 🎉 


### Accountability and Support Check

**open issues**
- [ ] #1143 
- [ ] #1268 
- [ ] #1142 

**open PRs**
- [ ] #1688 / #1727 
- [ ] #1719 / #1384 

**previous action items**
- [x] none 🎉 

## New Action items
- [ ] make new agenda

## Other Notes:
-
